lib/rails/generators/haml/component_generator.rb

```diff
@@ -7,9 +7,12 @@ module Generators
     class ComponentGenerator < Erb::Generators::ComponentGenerator
       source_root File.expand_path("templates", __dir__)
       class_option :sidecar, type: :boolean, default: false
+      # class_option :inline, type: :boolean, default: false
```

Comment:
Not sure if it's a good idea to redefine the `class_option` as done above for `sidecar`, because it's already defined in the parent `Erb::Generators::ComponentGenerator`.

Reply:
I found this relevant discussion: https://github.com/github/view_component/pull/190#pullrequestreview-345518807 but maybe it's time to reconsider this?